index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Kanu Mike C</title>
+  </head>
+  <body>
+    <div class="hero">
+      <header>
+        <div class="logo">My Logo</div>
+        <nav>
+          <a href="#" class="Ham-menu">
+            <img src="./images/Union.png" alt="Ham buger menu" />
+          </a>
+          <div class="desktop-menu">
+            <ul>
+              <li></li>
+              <li></li>
+              <li></li>
+              <li></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <div class="banner">
```

Comment:
- [ ] Please kindly use the appropriate tags for each element. For example, by using `<section>` instead of `<div>` tag for the container you follow [HTML, CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).